src/pages/Checklist/index.tsx

```diff
@@ -123,18 +124,22 @@ export const Checklist = () => {
                 ensure the network stays as decentralised as possible.
               </Text>
             </li>
-            <ul>
-              <li>
-                <Text>
-                  For example, you need at least ~300 GB SSD to run geth fast
-                  sync on <i>mainnet</i> in November 2020, assuming you move the{' '}
-                  <i>ancient</i> directory on a different drive. Have in mind
-                  that geth <i>chaindata</i> grows with ~1 GB per day, so you
-                  should account for enough space on your SSD and HDD until you
-                  run maintenance on the node.
-                </Text>
-              </li>
-            </ul>
+            <li className="py5">
+              <Text>
+                For example, you need at least ~300 GB SSD to run Geth sync on{' '}
+                <i>mainnet</i> in November 2020, assuming you move the{' '}
+                <i>ancient</i> directory on a different drive. Have in mind that
+                Geth <i>chaindata</i> grows with ~1 GB per day, so you should
+                account for enough space on your SSD and HDD until you run
+                maintenance on the node.
+              </Text>
+            </li>
+            <li className="py5">
+              <Text>
+                <strong>Note:</strong> it may take day(s) to sync the Eth1
+                client in the worst-case scenario.
+              </Text>
+            </li>
           </ul>
         </section>
         <section>
```

Comment:
I think we should purge this keystore section. There is not much a user can do if that have performed these checklist items, if they have made deposits, then they don't really matter.